src/opossum_lib/cli.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: 2023 TNG Technology Consulting GmbH <https://www.tngtech.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import logging
+import sys
+from pathlib import Path
+
+import click
+from spdx.model.document import Document
```

Comment:
Optional idea: You could do
```suggestion
from spdx.model.document import Document as SpdxDocument
```
to clarify the naming as we now have spdx documents as well as opossum documents.

Reply:
Done